### PR TITLE
[Prowlarr] tips for Prowlarr

### PR DESCRIPTION
**Pull request by TRaSH-**
NEW: Tips for Prowlarr

### CODE REVIEW

File: docs/Prowlarr/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## Master
+
+Prowlarr does not yet have a stable release.
+
+## Develop
+
+![develop](https://img.shields.io/badge/dynamic/json?query=%24.version&url=https://raw.githubusercontent.com/hotio/prowlarr/testing/VERSION.json&label=Current%20Develop/Beta%20Version&style=for-the-badge&color=4051B5)
+
+(Beta): This is the testing edge. Released after tested in nightly to ensure no immediate issues. New features and bug fixes released here first after nightly. It can be considered semi-stable, but is still `beta`.
```
Comment (thezak48) [inline docs/Prowlarr/index.md:17]: Should be `Released after being tested in nightly`

File: docs/Prowlarr/index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    **You may not be able to go back to `master` after switching to this branch.**
+
+On GitHub, this is a snapshot of the `develop` branch at a specific point in time.
+
+## Nightly
+
+![nightly](https://img.shields.io/badge/dynamic/json?query=%24.version&url=https://raw.githubusercontent.com/hotio/prowlarr/nightly/VERSION.json&label=Current%20Nightly/Unstable%20Version&style=for-the-badge&color=4051B5)
+
+(Alpha/Unstable) : This is the bleeding edge. It is released as soon as code is committed and passes all automated tests. This build may have not been used by us or other users yet. There is no guarantee that it will even run in some cases. This branch is only recommended for advanced users. Issues and self investigation are expected in this branch.
```
Comment (thezak48) [inline docs/Prowlarr/index.md:31]: Grammar Fixup `This build may not have been used by us`

File: docs/Prowlarr/index.md (hunk 1 above)
Comment (TRaSH-) [reply docs/Prowlarr/index.md:17]: https://wiki.servarr.com/prowlarr/faq#how-do-i-update-prowlarr
image

File: docs/Prowlarr/index.md (hunk 2 above)
Comment (TRaSH-) [reply docs/Prowlarr/index.md:31]: https://wiki.servarr.com/prowlarr/faq#how-do-i-update-prowlarr
image

**Pull request merged**